habitat_baselines/rl/models/rnn_state_encoder.py

```diff
@@ -266,6 +266,10 @@ class RNNStateEncoder(nn.Module):
     timesteps to handle episodes ending in the middle of a rollout.
     """
 
+    def __init__(self):
+        super().__init__()
+        self.rnn: torch.nn.Module
```

Comment:
This doesn't need to be in the init, it can just be declared in the class level namespace
`rnn: torch.nn.Module`